### PR TITLE
allow saving and loading exp. cov.

### PR DESCRIPTION
**Pull request by peterstangl**
@DavidMStraub do you think it makes sense to implement saving and loading of experimental covariance matrices like this?

### CODE REVIEW

File: smelli/classes.py [hunk 1]
```diff
@@ -155,6 +161,16 @@ def load_sm_covariances(self, folder):
             filename = os.path.join(folder, name + '.p')
             flh.sm_covariance.load(filename)
 
+    def save_exp_covariances(self, folder):
```
Comment (DavidMStraub) [inline smelli/classes.py:164]: Can you please add a doc string?

**Comment by DavidMStraub** [comment]
Yes this is certainly the best way to implement it. I'm not sure this is needed by many people, but it certainly doesn't hurt. What we should  probably not do is to have a precomputed covariance like for the SM covariance, as we want to keep making changes to measurements very easy. (Also it is easier to maintain if we only have to recompute SM covariances every time.)

File: smelli/classes.py (hunk 1 above)
Comment (peterstangl) [reply smelli/classes.py:164]: Like for `save_sm_covariances`, this method is mentioned in the doc string of `GlobalLikelihood` (in line 42). Don't you think that's enough?

**Comment by peterstangl** [comment]
> What we should probably not do is to have a precomputed covariance like for the SM covariance, as we want to keep making changes to measurements very easy. (Also it is easier to maintain if we only have to recompute SM covariances every time.)

Yes, I fully agree. That's also a reason why one has to explicitly specify a folder from which the experimental covariances are loaded and by default they are not loaded.

**Pull request merged**